test/e2e/network_segmentation_services.go

```diff
@@ -239,11 +287,32 @@ var _ = Describe("Network Segmentation: services", func() {
 				Expect(err).NotTo(HaveOccurred())
 
 				By("Verify the UDN client connection to the default network service")
+				checkNoConnectionToLoadBalancers(f, udnClientPod2, defaultService)
 				checkConnectionToNodePort(f, udnClientPod2, defaultService, &nodes.Items[0], "server node", defaultServerPod.Name)
 				checkNoConnectionToNodePort(f, udnClientPod2, defaultService, &nodes.Items[1], "local node")
 				checkConnectionToNodePort(f, udnClientPod2, defaultService, &nodes.Items[2], "other node", defaultServerPod.Name)
 				checkNoConnectionToClusterIPs(f, udnClientPod2, defaultService)
 
+				By("Creating a default network ExternalIPs service")
+				defaultExternalIPsService := &v1.Service{
+					ObjectMeta: metav1.ObjectMeta{Name: "service-default-external-ips"},
+					Spec: v1.ServiceSpec{
+						Ports: []v1.ServicePort{
+							{
+								Name:       "udp-port",
+								Port:       81,
+								TargetPort: intstr.FromInt(81),
+								Protocol:   v1.ProtocolUDP,
+							},
+						},
+						Selector:       defaultLabels,
+						ExternalIPs:    internalIPsFromNodes(nodes.Items),
+						IPFamilyPolicy: &policy,
+					},
+				}
+				By("Verify the UDN client connection to the default network external IPs service has no isolation")
+				checkNoConnectionToExternalIPs(f, udnClientPod2, defaultExternalIPsService, defaultExternalIPsService.Spec.ExternalIPs)
```

Comment:
Wait, UDN pod to an external IP that is not the same as its node IP should work even if the service is in the default network.

Reply:
Something like this ? 
```golang
 By("Verify isolation between UDN client and the default network external IPs service")                
                checkConnectionToExternalIPs(f, udnClientPod2, defaultExternalIPsService, internalIPsFromRemoteNode(nodes.Items, udnClientPod2))
                checkNoConnectionToExternalIPs(f, udnClientPod2, defaultExternalIPsService, internalIPsFromLocalNode(nodes.Items, udnClientPod2))
```

Reply:
yup, exactly